### PR TITLE
Add an emit function to browser sockets.

### PR DESCRIPTION
**Pull request by UnwrittenFun**
Currently an error is thrown on frontend due to the heartbeats using the emit method.

### CODE REVIEW

File: lib/socketMaker.js [hunk 1]
```diff
@@ -31,6 +31,19 @@ function wrapDOM(socket) {
         });
     };
 
+    socket.emit = function (event, ...data) {
```
Comment (connor4312) [inline lib/socketMaker.js]: The socket is not transpiled, ES5 only please!

**Comment by connor4312** [comment]
Looks good after the es6 part is removed

File: lib/socketMaker.js (hunk 1 above)
Comment (UnwrittenFun) [inline lib/socketMaker.js]: Er... I would hope it is transpiled, considering all the arrow functions and the `export default`..?

Comment (connor4312) [inline lib/socketMaker.js]: Ah, sorry, was thinking this was `beam-client-node`. That works then 😛

File: lib/socketMaker.js [hunk 2]
```diff
@@ -30,9 +30,9 @@ function wrapDOM(socket) {
 
     socket.once = function (event, listener) {
         const wrapped = wrapHandler(event, listener);
-        socket.addEventListener(event, ev => {
+        socket.addEventListener(event, function handler (ev) {
```
Comment (ProbablePrime) [inline lib/socketMaker.js]: Arrow fnction plz this is an es6 module.

**Comment by ProbablePrime** [comment]
lgtm, to be clear this is an es6 module. :)

File: lib/socketMaker.js (hunk 2 above)
Comment (UnwrittenFun) [inline lib/socketMaker.js]: it can't be because we need to use the name

Comment (ProbablePrime) [inline lib/socketMaker.js]: ignore me :D *stares at the wall*

**Comment by JamyDev** [comment]
lgtm

**Pull request merged**